sympy/matrices/common.py

```diff
@@ -2018,6 +2018,49 @@ def replace(self, F, G, map=False, simultaneous=True, exact=None):
         """
         return self.applyfunc(
             lambda x: x.replace(F, G, map=map, simultaneous=simultaneous, exact=exact))
+    
+    def rot90(self, anticlockwise = False):
+        """Rotates Matrix by 90 degrees
+
+        Examples
+        ========
+
+        >>> from sympy import Matrix
+        >>> M = Matrix(2, 2, lambda i, j: i+j)
```

Comment:
I think that you should give a better example than this which doesn’t have any symmetry.

I have noted about using a MatrixSymbol to show how the coordinates are transformed, rather than the values.

Another way can be using a matrix with unique decimal entries like 11, 12, 13, 21, 22 that can easily be thought as coordinates.

Reply:
Done ... I've changed these to MatrixSymbols